roles/kubernetes-apps/ansible/templates/kubedns-deploy.yml.j2

```diff
@@ -30,7 +30,7 @@ spec:
 {% if kube_version is version('v1.11.1', '>=') %}
       priorityClassName: system-cluster-critical
 {% endif %}
-      # When having win nodes in cluster without this patch, this pod cloud try to be created in windows
+      # When having win nodes in cluster without this patch, this pod could try to be created in windows
```

Comment:
same

Reply:
Since this exact same comment appears in a handful of places, I removed all occurrences.